1.3/Defs/Morphs/Animal/Avian/Penguin/Penguin_MorphDef.xml

```diff
@@ -1,5 +0,0 @@
-<Defs xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../Schemas/MorphDef.xsd">
```

Comment:
I don't know if we want to keep these penguin stub files since they are planned for implementation and we are already defining parts for them

Reply:
I thought I removed this.